backend/src/db/seed/seedScript.ts

```diff
@@ -0,0 +1,6 @@
+import { seedFakerSampleData } from '@backend/db/seed/fakeSampleData';
+import { getPrisma } from '@backend/db/client';
```

Comment:
you can use the synchronous prisma client getter here when merged - #30 
don't really need  to use getPrisma()